circuit_knitting/cutting/qpd/qpd.py

```diff
@@ -215,6 +215,16 @@ def qpdbasis_from_gate(gate: Gate) -> QPDBasis:
         return f(gate)
 
 
+def supported_gates() -> set[str]:
+    """
+    Return a set of gate names supported for automatic decomposition.
+
+    Returns:
+        Set of gate names supported for automatic decomposition.
+    """
+    return {"rxx", "ryy", "rzz", "crx", "cry", "crz", "cx", "cz"}
```

Comment:
```suggestion
    return set(_qpdbasis_from_gate_funcs)
```

I tested this locally, and it works.  Doing this the [DRY](https://en.wikipedia.org/wiki/Don%27t_repeat_yourself) way, as I suggest, will make sure this list does not go out of date.

Reply:
Thank you! I've been racking my brain trying to figure out how to make our test fail if a gate is added and it's not updated